get-dcos-docker.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Launches a DC/OS dev environment based containing a running 3-agent cluster and dev tools.
+#
+# Prerequisites:
+# - 8GB RAM
+# - Vagrant
+# - Virtualbox
+#
+# Arguments:
+# - BOX_PATH: Local path to use for .box image (useful if installing from USB stick)
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/create-dev-env-start.png >/dev/null 2>&1
+
+# abort script at first error:
+set -e
+
+BOX_NAME=mesosphere/dcos-docker-sdk
+
+error_msg() {
+    echo "---"
+    echo "Failed to build the cluster: Exited early at $0:L$1"
+    echo "To try again, re-run this script."
+    echo "---"
+}
+trap 'error_msg ${LINENO}' ERR
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR
+
+cd tools/vagrant/
+VAGRANT_DIR=$(pwd)
+
+# Image-on-USB-stick scenario: Manually add the provided .box file directly
+if [ $(vagrant box list | grep $BOX_NAME | wc -l) -eq 0 ]; then
+    if [ -n "$BOX_PATH" ]; then
+        echo "### Installing provided local box image as $BOX_NAME: $BOX_PATH"
+        vagrant box add --name $BOX_NAME ${BOX_PATH}
+    fi
```

Comment:
In that case `vagrant up` further down will automatically fetch the .box file referenced in the `Vagrantfile` before starting it:
- Looks at http://downloads.mesosphere.com/dcos-docker-sdk/metadata.json (or `DCOS_BOX_URL` if a custom one is set)
- Json in turn (currently) points to https://downloads.mesosphere.com/dcos-docker-sdk/dcos-docker-sdk-20161121-213020.box

This section is effectively an override of the default behavior, allowing the user to specify a different local copy of the .box file (hence USB-stick case)